api/api/models/audio.py

```diff
@@ -1,3 +1,5 @@
+from textwrap import dedent as d
```

Comment:
Kind of a nit, but the `d` alias is a bit obscure and roundabout compared to using the real name. What's the reason to alias this, does it aid in reading the code? I could see it being nice when writing, but after that, it's just a bit more confusing. Without the context of having just read this import, it looks kind of random in the code itself, and in the code itself I'd almost assume it was some kind of translation/gettext utility based on how similar single-letter string functions work in that space (including in our frontend).